src/Planners/include/BipedalLocomotion/Planners/SO3Planner.tpp

```diff
@@ -91,10 +215,12 @@ bool SO3Planner<trivialization>::evaluatePoint(const std::chrono::nanoseconds& t
     //                                                                                 Trivialization)
     // You can find the computation of s in "Modern Robotics: Mechanics, Planning, and Control"
     // (Chapter 9.2)
-    // TODO (Giulio) avoid to use t / T you may use std::chrono::ratio
-    const double s = 10 * std::pow(t / T, 3)
-                     - 15 * std::pow(t / T, 4)
-                     + 6 * std::pow(t / T, 5);
+    const double s = polynomial.a0 + //
+                     polynomial.a1 * t + //
+                     polynomial.a2 * t * t + //
+                     polynomial.a3 * t * t * t + //
+                     polynomial.a4 * t * t * t * t + //
+                     polynomial.a5 * t * t * t * t * t;
```

Comment:
I am a bit afraid of the numeric issues arising when multiplying these doubles together. 
For example, the order of evaluation is compiler dependent (https://en.cppreference.com/w/cpp/language/eval_order).

In other words, the output of ``a1*t*t`` might be different from ``a1*(t*t)``.

What about reusing the subexpressions. Something like:
```
t2 = t*t;
t3 = t2*t;
t4 =  t3*t;
t5 = t4*t;
s= a0 + a1 * t + a2*t2 + ....
```

Reply:
Done here: https://github.com/ami-iit/bipedal-locomotion-framework/pull/688/commits/7b328423092a305b5f4c506e37a1a9bd1657cb70